tensilelite/Tensile/SolutionStructs.py

```diff
@@ -2009,6 +2009,12 @@ def assignDerivedParameters(state):
       state["1LDSBuffer"] = 1
       print2("\nSet SIA=2, force PrefetchLocalRead=1, ExpandPointerSwap=1, 1LDSBuffer=1")
 
+    if not globalParameters["AsmCaps"][isa]["HasNTModifier"]:
```

Comment:
I think this needs to be done after resolving state["NonTemporal"] != -1.
Otherwise, it can be overwritten by state["NonTemporal"].

Reply:
Sorry, I missed "".

Reply:
I think it works as written as well. "NonTemporal" is included in the loop with the empty string "", so if "NonTemporal" is set to an invalid values, it will get updated before it gets copied to the other settings.